src/gui/guiConfirmRegistration.cpp

```diff
@@ -96,8 +96,14 @@ void GUIConfirmRegistration::regenerateGui(v2u32 screensize)
 				"Join to confirm account creation or click Cancel to "
 				"abort.");
 		char info_text_buf[1024];
+
+#ifdef _WIN32
```

Comment:
since this doesn't happen on mingw, it should be `#ifdef _MSC_VER` instead

Reply:
Yes you are right, thanks